index.js

```diff
@@ -116,14 +126,13 @@ module.exports = function (homebridge) {
     }
 
     getServices() {
-
       // TODO
       let informationService = new homebridge.hap.Service.AccessoryInformation()
         .setCharacteristic(homebridge.hap.Characteristic.Manufacturer, 'Pilight Manufacturer')
         .setCharacteristic(homebridge.hap.Characteristic.Model, 'Pilight Model')
         .setCharacteristic(homebridge.hap.Characteristic.SerialNumber, 'Pilight Serial Number');
 
-      let lightbulbService = new homebridge.hap.Service.Lightbulb();
```

Comment:
I think Switch is much better than LightBulb (I'm not aware of the other types). But then the rest should be renamed also. I don't want to fool the (code) reader.